conf/base.config

```diff
@@ -9,15 +9,13 @@
  * run on the logged in environment.
  */
 
-process {
 
-  cpus = { check_max( 1 * task.attempt, 'cpus' ) }
-  memory = { check_max( 7.GB * task.attempt, 'memory' ) }
-  time = { check_max( 4.h * task.attempt, 'time' ) }
+process {
 
```

Comment:
Modification to avoid AWS issue when !awsbatch

```console
WARN: ### Task (id=i) requests an amount of resources not available in any node in the current cluster topology 
```